liwords-ui/src/gameroom/board_panel.tsx

```diff
@@ -431,6 +438,7 @@ export const BoardPanel = React.memo((props: Props) => {
       setArrowProperties({ row: 0, col: 0, horizontal: false, show: false });
     }
     lastLettersRef.current = props.board.letters;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
So for this particular one, do we just live with the warning?

Reply:
as @andy-k explained, useState setters are exempt from needing to be listed in the requirements, but we should add these three to the requirements: ```    setPlacedTilesTempScore,
    setPlacedTiles,
    setDisplayedRack,```

Reply:
In this case, the missing dependencies are the setters from the store. Not sure how to list those without causing infinite stack depth. Do you have any examples of calling in those stores in a more static way so that doesn't happen?

Reply:
thanks for uncovering this bug. #230 should make it work